inc/providers/class-core-sitemaps-posts.php

```diff
@@ -68,7 +68,20 @@ public function get_url_list( $page_num, $post_type = '' ) {
 			return array();
 		}
 
-		$query = new WP_Query(
+		/**
+		 * Filters the post type URL list query arguments.
+		 *
+		 * Allows modification of the URL list query arguments before querying.
+		 *
+		 * @see WP_Query for a full list of arguments
+		 *
+		 * @since 5.5.0
+		 *
+		 * @param array  $args      An array of WP_Query arguments.
+		 * @param string $post_type The post type string.
+		 */
+		$args = apply_filters(
+			'core_sitemaps_post_url_list_query_args',
```

Comment:
In accordance with other filter names (including the other one introduced here below), it'd be more consistent to use the "posts" plural here (`core_sitemaps_posts_url_list_query_args`).